commit 7e07f311aa358021ca5b3940fd4bd3aae8b9c6e5
Author: Trent Mick <trentm@gmail.com>
Date:   2019-04-16T14:42:49-07:00 (6 months ago)
    
    TRITON-1401 rip out "standalone" mode of running cloudapi tests
    Reviewed by: Marsell Kukuljevic <marsell@joyent.com>
    Approved by: Marsell Kukuljevic <marsell@joyent.com>

diff --git a/Makefile b/Makefile
index 9255e2622..ffe6435d0 100644
--- a/Makefile
+++ b/Makefile
@@ -5,7 +5,7 @@
 #
 
 #
-# Copyright (c) 2019, Joyent, Inc.
+# Copyright 2019 Joyent, Inc.
 #
 
 #
@@ -152,80 +152,6 @@ publish: release
 	mkdir -p $(ENGBLD_BITS_DIR)/$(NAME)
 	cp $(ROOT)/$(RELEASE_TARBALL) $(ENGBLD_BITS_DIR)/$(NAME)/$(RELEASE_TARBALL)
 
-.PHONY: test no_machines_test auth_test account_test datacenters_test datasets_test fabrics_test images_test keys_test networks_test nics_test machines_all_test machines_70_test machines_71_test machines_72_test machines_73_test machines_80_test machines_test packages_test populate_networks_test services_test users_test provision_limits_plugin_test plugins_test tests_test
-
-auth_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/auth.test.js
-
-account_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/account.test.js
-
-datacenters_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/datacenters.test.js
-
-datasets_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/datasets.test.js
-
-fabrics_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/fabrics.test.js
-
-images_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/images.70.test.js
-	$(NODE_EXEC) $(TAP) test/images.80.test.js
-	$(NODE_EXEC) $(TAP) test/images.test.js
-
-keys_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/keys.test.js
-
-networks_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/networks.test.js
-
-nics_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/nics.test.js
-
-machines_all_test:
-	$(NODE_EXEC) $(TAP) test/machines.test.js
-
-machines_70_test:
-	$(NODE_EXEC) $(TAP) test/machines.70.test.js
-
-machines_71_test:
-	$(NODE_EXEC) $(TAP) test/machines.71.test.js
-
-machines_72_test:
-	$(NODE_EXEC) $(TAP) test/machines.72.test.js
-
-machines_73_test:
-	$(NODE_EXEC) $(TAP) test/machines.73.test.js
-
-machines_80_test:
-	$(NODE_EXEC) $(TAP) test/machines.80.test.js
-
-machines_test: machines_all_test machines_70_test machines_71_test machines_72_test machines_73_test machines_80_test
-
-packages_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/packages.test.js
-
-populate_network_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/populate_network.test.js
-
-services_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/services.test.js
-
-tests_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/tests.test.js
-
-users_test: $(TAP)
-	$(NODE_EXEC) $(TAP) test/users.test.js
-
-test: auth_test account_test datacenters_test datasets_test fabrics_test images_test keys_test networks_test packages_test populate_network_test services_test tests_test users_test nics_test machines_test
-
-no_machines_test: auth_test account_test datacenters_test datasets_test fabrics_test images_test keys_test networks_test packages_test populate_network_test services_test tests_test users_test
-
-provision_limits_plugin_test:
-	$(NODE_EXEC) $(TAP) test/provision_limits.test.javascript
-
-plugins_test: provision_limits_plugin_test
 
 include ./deps/eng/tools/mk/Makefile.deps
 ifeq ($(shell uname -s),SunOS)
diff --git a/test/common.js b/test/common.js
index c3b46dd41..bfefb00b5 100644
--- a/test/common.js
+++ b/test/common.js
@@ -32,7 +32,6 @@ var PAPI = require('sdc-clients').PAPI;
 var MAHI = require('mahi');
 var VOLAPI = require('sdc-clients').VOLAPI;
 
-var app = require('../lib').app;
 var apertureConfig = require('aperture-config').config;
 
 
@@ -503,24 +502,11 @@ function setupClient(version, serverUrl, user, keyId, keyPath, parentAcc, cb) {
 
 
 function loadServer(cb) {
-    if (process.env.SDC_SETUP_TESTS) {
-        var serverObj = {
-            url: process.env.SDC_SETUP_URL || 'https://127.0.0.1'
-        };
-
-        return cb(null, serverObj);
-    }
-
-    return app.createServer(CONFIG, function (err, server) {
-        if (err) {
-            return cb(err);
-        }
+    var serverObj = {
+        url: 'https://127.0.0.1'
+    };
 
-        return server.start(function () {
-            LOG.info('CloudAPI listening at %s', server.url);
-            cb(null, server);
-        });
-    });
+    cb(null, serverObj);
 }
 
 
diff --git a/test/runtests b/test/runtests
index 70f852944..2ffe8bd04 100755
--- a/test/runtests
+++ b/test/runtests
@@ -112,7 +112,7 @@ failed_test_names=
 set +o errexit
 set -o pipefail
 for t in ${test_files}; do
-    PATH=${NODE_INSTALL}/bin:${PATH} SDC_SETUP_TESTS=1 ${NODE_INSTALL}/bin/node \
+    PATH=${NODE_INSTALL}/bin:${PATH} ${NODE_INSTALL}/bin/node \
         --abort_on_uncaught_exception ${t} | tee -a ${OUTPUT_DIR}/cloudapi.tap
     if [[ $? != 0 ]]; then
         failed_test_names="${failed_test_names} ${t}"
